package.json

```diff
@@ -32,6 +32,7 @@
     "@nestjs/swagger": "^4.7.12",
     "axios": "^0.21.2",
     "bcrypt": "^5.0.0",
+    "class-transformer": "^0.4.0",
```

Comment:
@Kif-Fando when you install a new package, you should commit both package.json and package-lock.json files

Reply:
Oh, my fault

Reply:
Never mind, I have already fixed that